virttest/qemu_storage.py

```diff
@@ -127,13 +128,24 @@ def create(self, params, ignore_errors=False):
             qemu_img_cmd = qemu_img_cmd.rstrip(",")
 
             if self.base_tag:
+                if params.get("image_json_file.driver"):
+                    jdict = {}
+                    keys = [key for key in params.keys()
+                            if key.startswith('image_json')]
+                    for key in keys:
```

Comment:
why not "file.driver_image_json" so that you can get insersting params  by "params.object_params()"